client/src/components/Accordions.js

```diff
@@ -89,10 +89,44 @@ const StatelessPullDownAccordion = ({
   isExpanded,
   children,
   onToggle,
+  showPin,
+  sticky_data, //sticky data only applies if show pin is true
```

Comment:
This is an old component that used inconsistent prop type casing. `show_pin` would be more in-line with current code style.

I'd question grouping the `can_pin` and the `onClick` callback inside `sticky_data`. It obscures what that prop actually is (you have to look through render and see what properties are accessed from it). Plus, it's arguably not really data.